### PR TITLE
Data Visualization tutorial - 2nd half static map

### PR DESCRIPTION
**Pull request by weiji14**
Placeholder for the Data Visualization tutorial on Day 3 - Wednesday 23 March 10:45 - 11:45 (PST).

Leads: @fliphilipp, @weiji14
Helpers: @wsauthoff

**Preview** tutorial at https://deploy-preview-40--icesat2-2022.netlify.app/tutorials/datavisualization/dataviz2d

Suggestions are more than welcome :)

### CODE REVIEW

File: book/_config.yml [hunk 1]
```diff
@@ -67,6 +67,10 @@ bibtex_bibfiles:
 sphinx:
   config:
     bibtex_reference_style: author_year  # or label, super, \supercite
+    nb_custom_formats:
+        .py:
+            - jupytext.reads
+            - fmt: py:percent
```
Comment (scottyhq) [inline book/_config.yml:73]: this is awesome. I think I'm going to start using jupytext from here on out ;)

**Comment by scottyhq** [comment]
I was hoping the s3 access would work, but then remembered we have to be on an AWS us-west-2 instance and github actions runs on Azure...
```
# Login to Earthdata and download the ATL14 NetCDF file using icepyx
region_iceland.earthdata_login(
    uid="uwhackweek",  # EarthData username, e.g. penguin123
    email="hackweekadmin@uw.edu",  # e.g. penguin123@southpole.net
    s3token=True,
)
region_iceland.download_granules(path="/tmp")
```
```pytb
AssertionError: You must be working from a valid AWS instance to use s3 data access
```

**Review by scottyhq — APPROVED** [review]
This is fantastic, thanks @weiji14 ! Note that I made a small change to your data download cell in order to fully execute the notebook vi CI, most of time is now spent downloading data of course. I also made a copy of the datasets that get downloaded at `~/shared/dataviz`: 

```
ATL14_GL_0311_100m_001_01.nc          ATL14_IS_0311_100m_001_01.nc.iso.xml
ATL14_GL_0311_100m_001_01.nc.iso.xml  processed_ATL11_135803_0311_004_01.h5
ATL14_IS_0311_100m_001_01.nc
```

Feel free to open a new PR if you'd like to make any further changes

File: book/_config.yml (hunk 1 above)
Comment (weiji14) [reply book/_config.yml:73]: Yes, it's much easier to review on GitHub than a JSON file!

**Comment by weiji14** [comment]
> This is fantastic, thanks @weiji14 ! Note that I made a small change to your data download cell in order to fully execute the notebook vi CI, most of time is now spent downloading data of course. I also made a copy of the datasets that get downloaded at `~/shared/dataviz`: 
> 
> ```
> ATL14_GL_0311_100m_001_01.nc          ATL14_IS_0311_100m_001_01.nc.iso.xml
> ATL14_GL_0311_100m_001_01.nc.iso.xml  processed_ATL11_135803_0311_004_01.h5
> ATL14_IS_0311_100m_001_01.nc
> ```
> 
> Feel free to open a new PR if you'd like to make any further changes

Cool, I actually commented the ATL14 download cell out because icepyx grabs Greenland even though I just need Iceland. But ok to have both on the shared drive in case someone wants to look at Greenland.

**Pull request merged**